tests/parametrize_with_checks_encoders_v16.py

```diff
@@ -0,0 +1,54 @@
+"""
+File intended to help understand check_estimator tests for Feature-engine's
+encoding module. It is not run as part of the battery of acceptance tests.
+Works from sklearn > 1.6.
+"""
+
+from sklearn.utils.estimator_checks import parametrize_with_checks
+
+from feature_engine.encoding import (
+    CountFrequencyEncoder,
+    MeanEncoder,
+    OneHotEncoder,
+    OrdinalEncoder,
+    RareLabelEncoder,
+    StringSimilarityEncoder,
+    WoEEncoder,
+)
+from feature_engine.tags import _return_tags
+
+ce = CountFrequencyEncoder(ignore_format=True)
+me = MeanEncoder(ignore_format=True)
+ohe = OneHotEncoder(ignore_format=True)
+oe = OrdinalEncoder(ignore_format=True)
+re = RareLabelEncoder(
+    tol=0.00000000001,
+    n_categories=100000000000,
+    replace_with=10,
+    ignore_format=True,
+)
+woe = WoEEncoder(ignore_format=True)
+sse = StringSimilarityEncoder(ignore_format=True)
+
+FAILED_CHECKS = _return_tags()["_xfail_checks"]
+FAILED_CHECKS.update({"check_estimators_nan_inf": "transformer allows NA"})
```

Comment:
Hey @solegalli, would't it make more sense to add this failing check as an extra tag of the classes we are testing in this file?

Reply:
What do you mean?